x-pack/plugins/features/kibana.json

```diff
@@ -6,5 +6,6 @@
   "optionalPlugins": ["visTypeTimelion"],
   "configPath": ["xpack", "features"],
   "server": true,
-  "ui": true
+  "ui": true,
+  "extraPublicDirs": ["common"]
```

Comment:
It exports only types, don't think it's necessary. Do you want to include `common` always? `src/plugins/saved_objects_management/kibana.json` doesn't have one.

Reply:
> It exports only types, don't think it's necessary. 

It exports the [`Feature` class](https://github.com/elastic/kibana/blob/master/x-pack/plugins/features/common/feature.ts#L131)

> Do you want to include `common` always? `src/plugins/saved_objects_management/kibana.json` doesn't have one.

I would rather make it explicit when it's necessary/desired for common to be included in the browser bundles. It seems reasonable to me that code shouldn't be in a `common` directory unless it's used by the browser, but it doesn't seem to be the case right now so I'd rather not inflate the bundles more than necessary.

Reply:
hm. it's used a class only within the `Features` plugin. ok, let's keep it as a temporary measurement. Interesting to see what's left in the final build.